### PR TITLE
Test FirewallD

### PR DESCRIPTION
**Pull request by atanno**
FirewallD basic functionality test.

- Related ticket: https://progress.opensuse.org/issues/47159
- Verification run: http://10.100.51.204/tests/60#

### CODE REVIEW

**Comment by asmorodskyi** [comment]
Can please squash commits into one ? Also please read https://chris.beams.io/posts/git-commit/ to come up with better commit message

File: tests/console/firewalld.pm [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+use warnings;
+use base "consoletest";
+use testapi;
+
+sub run {
+    select_console("root-console");
+
+    # Check Service State, enable it if necessary
+    record_info 'Check Service State';
+    #assert_script_run("if [ `firewall-cmd --state > /dev/null 2>&1 ; echo $?` -eq 252 ]; then systemctl start firewalld; fi");
```
Comment (asmorodskyi) [inline tests/console/firewalld.pm]: delete this line

File: tests/console/firewalld.pm [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    record_info 'Test #6', 'Change the default zone';
+    assert_script_run("firewall-cmd --set-default-zone=public");
+    assert_script_run("if [ a`firewall-cmd --get-default-zone | grep -i 'public'` = \"apublic\" ]; then true; else false; fi");
+    assert_script_run("firewall-cmd --set-default-zone=dmz");
+    assert_script_run("if [ a`firewall-cmd --get-default-zone | grep -i 'dmz'` = \"admz\" ]; then true; else false; fi");
+
+    # Change to the default zone
+    record_info 'Set Default Zone';
+    assert_script_run("firewall-cmd --set-default-zone=public");
+    assert_script_run("if [ a`firewall-cmd --get-default-zone | grep -i 'public'` = \"apublic\" ]; then true; else false; fi");
```
Comment (mimi1vx) [inline tests/console/firewalld.pm]: `firewall-cmd --get-default-zone | grep -q  -i 'public'` instead `if then else` construction is sufficient grep returns 0 if is  pattern found else 1

+ same thing for other commands

File: tests/console/firewalld.pm [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    select_console("root-console");
+
+    # Check Service State, enable it if necessary
+    record_info 'Check Service State';
+    #assert_script_run("if [ `firewall-cmd --state > /dev/null 2>&1 ; echo $?` -eq 252 ]; then systemctl start firewalld; fi");
+    assert_script_run("if [ `systemctl is-active firewalld > /dev/null 2>&1 ; echo $?` -eq 0 ]; then systemctl start firewalld; fi");
+
+    # Test #1 - Stop firewalld then start it
+    record_info 'Test #1', 'Test: Stop firewalld, then start it';
+    assert_script_run("systemctl stop firewalld");
```
Comment (asmorodskyi) [inline tests/console/firewalld.pm]: there is wrapper function in openQA for systemctl call :
```suggestion
    systemctl "stop firewalld";
```

Comment (atanno) [reply tests/console/firewalld.pm]: systemctl("stop firewalld");

**Pull request closed without merging**